mod_reforged/hooks/states/tactical_state.nut

```diff
@@ -1,4 +1,30 @@
 ::Reforged.HooksMod.hook("scripts/states/tactical_state", function(q) {
+	q.initMap = @(__original) function()
+	{
+		// Feat: the same location will always produce the exact same layout
+		// This has no influence over troop spawning as those will be seeded  by "combatSeed" later on
+		if (this.m.StrategicProperties.MapSeed == 0)	// Other fixed seeds take priority
```

Comment:
In the original function I have seen that it also checks for `LocationTemplate != null` in addition to `MapSeed != 0` before it does the `seedRandom`. Do we not need that condition as well here?

Reply:
I dont remember why I only went with one check.
Upon further looking at the vanilla code I now chose to remove all condition checks, as our code runs before the vanilla code anyways. They will overwrite our seeding attempt.
And the performance impact of not checking the condition first is negligible, as this runs only once per combat initialization